pr_agent/agent/pr_agent.py

```diff
@@ -16,16 +16,16 @@ def __init__(self):
     async def handle_request(self, pr_url, request) -> bool:
         action, *args = request.strip().split()
         if any(cmd == action for cmd in ["/answer"]):
-            await PRReviewer(pr_url, is_answer=True).review()
+            await PRReviewer(pr_url, is_answer=True, args=args).review()
         elif any(cmd == action for cmd in ["/review", "/review_pr", "/reflect_and_review"]):
             if settings.pr_reviewer.ask_and_reflect or "/reflect_and_review" in request:
-                await PRInformationFromUser(pr_url).generate_questions()
+                await PRInformationFromUser(pr_url, args=args).generate_questions()
             else:
                 await PRReviewer(pr_url, args=args).review()
         elif any(cmd == action for cmd in ["/describe", "/describe_pr"]):
             await PRDescription(pr_url, args=args).describe()
         elif any(cmd == action for cmd in ["/improve", "/improve_code"]):
-            await PRCodeSuggestions(pr_url).suggest()
+            await PRCodeSuggestions(pr_url, args=args).suggest()
         elif any(cmd == action for cmd in ["/ask", "/ask_question"]):
             await PRQuestions(pr_url, args=args).answer()
         elif any(cmd == action for cmd in ["/update_changelog"]):
```

Comment:
**Suggestion:** The `handle_request` function has a lot of if-else statements, which can be hard to maintain and extend. Consider using a dictionary to map actions to functions. This will make the code cleaner and easier to extend in the future.
```suggestion
    async def handle_request(self, pr_url, request) -> bool:
        action, *args = request.strip().split()
        actions = {
            '/answer': PRReviewer(pr_url, is_answer=True, args=args).review,
            '/review': PRReviewer(pr_url, args=args).review,
            '/review_pr': PRReviewer(pr_url, args=args).review,
            '/reflect_and_review': PRInformationFromUser(pr_url, args=args).generate_questions if settings.pr_reviewer.ask_and_reflect else PRReviewer(pr_url, args=args).review,
            '/describe': PRDescription(pr_url, args=args).describe,
            '/describe_pr': PRDescription(pr_url, args=args).describe,
            '/improve': PRCodeSuggestions(pr_url, args=args).suggest,
            '/improve_code': PRCodeSuggestions(pr_url, args=args).suggest,
            '/ask': PRQuestions(pr_url, args=args).answer,
            '/ask_question': PRQuestions(pr_url, args=args).answer,
            '/update_changelog': self.update_changelog,
        }
        if action in actions:
            await actions[action]()
```